### PR TITLE
kola/tests/ignition: add tests for ext4

### PR DESCRIPTION
**Pull request by ajeddeloh**
Add tests for creating ext4 filesystems in ignition. Test both creating
a new ext4 filesystem in ignition and tests not overwriting an existing
filesystem.

### CODE REVIEW

File: kola/tests/ignition/v2/root.go [hunk 1]
```diff
@@ -111,6 +168,13 @@ func xfsRoot(c platform.TestCluster) error {
 	return testRoot(c, "xfs")
 }
 
+func ext4Root(c platform.TestCluster) error {
+	//since the drive is formatted by default to ext4, this wont test if
```
Comment (crawford) [inline kola/tests/ignition/v2/root.go]: This comment is confusing.

File: kola/tests/ignition/v2/root.go [hunk 2]
```diff
@@ -125,3 +189,20 @@ func testRoot(c platform.TestCluster, fs string) error {
 
 	return nil
 }
+
+func ext4CheckExisting(c platform.TestCluster) error {
+	m := c.Machines()[0]
+
+	//redirect /dev/null to stdin so isatty(stdin) fails and the -p flag can be
```
Comment (crawford) [inline kola/tests/ignition/v2/root.go]: NIT: space between `//` and comment and standard English sentence.

**Comment by ajeddeloh** [comment]
bump

**Comment by crawford** [comment]
LGTM

**Pull request merged**